resources/processor-types/log_deduplicate.yaml

```
@@ -0,0 +1,37 @@
+apiVersion: bindplane.observiq.com/v1
+kind: ProcessorType
+metadata:
+  name: log_deduplicate
+  displayName: DeDuplicate Logs 
+  description: Aggregates logs over a specified time period and removes duplicates.
+spec:
+  version: 0.0.1
+  parameters:
+    - name: interval
+      label: Interval
+      description: The time interval (in seconds) over which to deduplicate logs.
+      type: int
+      default: 10
+      required: true
+
+    - name: log_count_attribute
+      label: Count Attribute Name
+      description: The name of the attribute to store the count of logs deduplicated.
+      type: string
+      required: true
+      default: log_count
+
+    - name: timezone
+      label: Timezone
+      advancedConfig: true
+      description: Timezone is used to localize the observed timestamp attributes.
+      type: string
+      required: true
+      default: UTC
+
+  logs:
+    processors: |
+      - logdedup:
+          interval: {{ .interval }}s
+          timezone: {{ .timezone }}
+          log_count_attribute: {{ .log_count_attribute }}
```

This is something Dave noticed with my PR, we should protect against users inserting quotes and malforming the yaml
```suggestion
          timezone: {{ .timezone | quote }}
          log_count_attribute: {{ .log_count_attribute | quote }}
```

---

Cool I didn't realize this was a thing.